Dockerfile-build

```diff
@@ -0,0 +1,38 @@
+# This file was cloned from https://github.com/mengzyou/docker-hugo/blob/main/Dockerfile
+# It was moved to this repo to protect against upstream version upgrades 
+
+# Dockerfile to build hugo sites 
+# It reads the local directory as source files
+# It outputs to the /docs directory
+# 
+# Usage: 
+# cd <sqlitebrowser-website-directory>
+
+# Build container with: docker build -f Dockerfile-build -t hugo-build .
+# Build Hugo site with: docker run --rm -v $(pwd):/home/hugo/app hugo-build
+
+# Base image
+FROM alpine:3.16
```

Comment:
Should we bump that to the latest alpine?

Reply:
Yes.